axum/src/extract/rejection.rs

```diff
@@ -58,6 +58,14 @@ define_rejection! {
     pub struct MissingPathParams;
 }
 
+define_rejection! {
+    #[status = INTERNAL_SERVER_ERROR]
+    #[body = "No original uri was found for the matched route. This is a bug in axum. Please open an issue"]
```

Comment:
As noted previously, this might also be a programmer error if using the extractor outside of a `Router` but thats quite an edge case so I think this wording is fine.

Reply:
Maybe weaken the message just a small bit?

```suggestion
    #[body = "No original uri was found for the matched route. This is likely a bug in axum. Please open an issue"]
```

Reply:
Sure@